src/Workspaces/Core/Portable/Shared/TestHooks/FeatureAttribute.cs

```diff
@@ -31,6 +31,7 @@ internal static class FeatureAttribute
     public const string FindReferences = nameof(FindReferences);
     public const string SemanticSearch = nameof(SemanticSearch);
     public const string GenerateDocumentation = nameof(GenerateDocumentation);
+    public const string GenerateMethod = nameof(GenerateMethod);
```

Comment:
what is this from?  Note: 'generate method' is already a feature.  (the feature that suggesting generating a meethod when the user writes `Foo()`).